webapp/channels/src/components/admin_console/password_settings.tsx

```diff
@@ -13,8 +13,36 @@ import Setting from './setting';
 import SettingsGroup from './settings_group.jsx';
 import TextSetting from './text_setting';
 
```

Comment:
Hi @Koosha-Owji 
There a few lint issues with your changes. Could you please look at these?
[Lint Check](https://github.com/mattermost/mattermost-server/actions/runs/5041097321/jobs/9046682900?pr=23473)